src/utilities/dates.ts

```diff
@@ -89,3 +92,35 @@ export function formatDateTimeRelative(value: Date | string, comparedTo: Date |
 
   return `in ${formatted}`
 }
+
+export function formatDateTime(value: Date | string, formatTime = timeNumericShortFormat): string {
```

Comment:
Is this a relative format? Like "today at 6:45 pm"?

Reply:
Correct, I really want to clean up these utilities and give them some more descriptive names but that's a wider change than feels necessary here. Maybe this could be better named something like `formatDateTimeColloquial` or something? Not sold on that

Reply:
realizing its missing but `formatDateTime` would be the dateTime formate of `MMM do, yyyy at hh:mm a`. Can we rename this to formateDateTimeRelative? Which following the naming of our style guide.